components/tooltip/__tests__/tooltip.browser-test.jsx

```diff
@@ -77,14 +77,6 @@ describe('SLDSTooltip: ', function () {
 			);
 		});
 
-		it('renders the content as assistive text', () => {
-			const span = findRenderedDOMComponentWithClass(
-				rootNode,
-				'slds-assistive-text'
-			);
-			expect(span.textContent).to.equal('This is more info. blah blah.');
-		});
```

Comment:
Is there a reason this unit test was removed?

Reply:
`.slds-assistive-text` was removed.